src/android/java/com/pspdfkit/cordova/action/document/ShowDocumentAction.java

```diff
@@ -95,39 +95,87 @@ private PdfActivityConfiguration parseOptionsToConfiguration(@NonNull final JSON
           builder.autosaveEnabled((Boolean) value);
         } else if ("backgroundColor".equals(option)) {
           builder.backgroundColor(Color.parseColor((String) value));
-        } else if ("disableAnnotationList".equals(option) && ((Boolean) value)) {
-          builder.disableAnnotationList();
+        } else if ("disableAnnotationList".equals(option)) {
+          if (((Boolean) value)) {
+            builder.disableAnnotationList();
+          } else {
+            builder.enableAnnotationList();
+          }
```

Comment:
The previous implementation could not differentiate between a mismatch of the option name and a `false` value, which lead to exception due to the "catch unknown properties" code further below.